### PR TITLE
Create header component

### LINKED ISSUE #4: Header Component
As a user
i need a header
for a overview where i am

- [x] Title for every page and subpage
- [x] Import Oxanium font from google fonts in 300 and 800
- [x] https://fonts.google.com/specimen/Oxanium
- [x] Add font to global css variables
- [x] Props for thin and bold text
- [x] Create Notification Icon
- [x] Create Close Icon
- [x] Prop for the specific icon (Notifications, Close, none)

### PR DESCRIPTION
**Pull request by ChristianSeelemann**
Deployment: https://fortknights-component-h-df4qfx.herokuapp.com/storybook/?path=/story/component-header--notification

- Title for every page and subpage
- Import Oxanium font from google fonts
- Add font to global css variables
- Props for thin and bold text
- Prop for the specific icon (Notifications, Close or a specific jsx element)
- Create Notification Icon
- Create Close Icon
- Add onClick prop to all icon components

### CODE REVIEW

File: src/app/components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useHistory } from 'react-router';
+import Close from '../Icons/Close';
+import Notification from '../Icons/Notification';
+import styles from './Header.module.css';
+
+type HeaderProps = {
+  textthin: string;
+  textbold: string;
```
Comment (marwinburesch) [inline src/app/components/Header/Header.tsx]: butWhereIsTheCamelCase? :>

File: src/app/components/Icons/Items.tsx [hunk 2]
```diff
@@ -2,16 +2,18 @@ import React from 'react';
 
 type ItemsProps = {
   color: string;
+  onClick?: () => void;
 };
 
-export default function Items({ color }: ItemsProps): JSX.Element {
+export default function Items({ color, onClick }: ItemsProps): JSX.Element {
```
Comment (marwinburesch) [inline src/app/components/Icons/Items.tsx:8]: Are those the only props you need?

File: src/app/components/Icons/Notification.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+type NotificationProps = {
+  color: string;
+  activecolor: string;
```
Comment (marwinburesch) [inline src/app/components/Icons/Notification.tsx]: 🐫

**Review by marwinburesch — COMMENTED** [review]
There's a lot of changes in here, that I didn't anticipate. I like your icon components except, `Close_Icon.svg` and `Notification_Icon.svg`, because all the others are React components.
Nice work!

File: src/app/components/Icons/Items.tsx (hunk 2 above)
Comment (ChristianSeelemann) [reply src/app/components/Icons/Items.tsx:8]: All icons have only one place in the layout at the moment. When this is changing i will add height, width and other essential props to it.

**Pull request merged**